build-system/pr-check.js

```diff
@@ -398,7 +400,8 @@ function main(argv) {
           util.colors.cyan('test/integration'));
       command.cleanBuild();
       command.buildRuntimeMinified();
```

Comment:
Sorry about the multi-part review. I wonder if `validator_minified.js` is generated by `buildRuntimeMinified`, which calls `gulp dist --fortesting`. If so, we can avoid the call to `buildValidator` before calling `runIntegrationTests`.